### PR TITLE
Add doc about plugin registration.

### PR DESCRIPTION
**Pull request by rgwozdz**
Adding a specific doc about plugin registration requirements.

### CODE REVIEW

**Comment by rgwozdz** [comment]
No, my bad, thanks for catching.

**Comment by rgwozdz** [comment]
@haoliangyu - just pushed https://github.com/koopjs/koopjs.github.io/pull/70/commits/9202ee760be8d8d8ab1e9e0af91c41731121adda, which removes duplication.

File: _docs/development/registration.md [hunk 1]
```diff
@@ -42,9 +41,14 @@ module.exports = function(options = {}) {
 }
 ```
 
-Note that when registering a plugin that exposes a function instead of an object, you must execute the function when it is required so that it returns an object. Continuing on from the exported function example above:
+Lastly, some plugin types define constructor function with specific properties:
```
Comment (haoliangyu) [inline _docs/development/registration.md]: Can you specify the `some plugin types` here?

Comment (rgwozdz) [reply _docs/development/registration.md]: Let me know what you think about this: https://github.com/koopjs/koopjs.github.io/pull/70/commits/62b998910976c2634f38087b159775b2dcd0aa07.

Comment (haoliangyu) [reply _docs/development/registration.md]: Looks good :+1:

**Pull request merged**